ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/jenkins_user/tasks/main.yml

```diff
@@ -8,62 +8,77 @@
   register: jenkins
   tags: jenkins_user
 
-- debug:
+- name: debug1
+  debug:
     msg: "{{ Jenkins_Username }} home directory found, skipping user creation tasks"
   when: jenkins.stat.isdir is defined
   tags: jenkins_user
 
 - name: Create jenkins user
-  command: mkuser home="/home/{{ Jenkins_Username }}" shell="/bin/bash" {{ Jenkins_Username }}
-  ignore_errors: yes
-  when: jenkins.stat.isdir is not defined
-  tags: jenkins_user
+  block:
+    - name: Create jenkins user
+      command: mkuser home="/home/{{ Jenkins_Username }}" shell="/usr/bin/ksh" {{ Jenkins_Username }}
+      ignore_errors: yes
+      tags: jenkins_user
 
-- name: Create SSH Key folder for {{ Jenkins_Username }}
```

Comment:
Doesn't need to be changed now but we should look at doing this in the same we we do on the other UNIX platforms: https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml#L69

Reply:
I would like to make more changes - good discussion for a different PR.